### PR TITLE
Configuration and docker updates

### PR DESCRIPTION
**Pull request by hendricksond**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,13 +1,15 @@
-app:
+twitarr:
```
Comment (grundoon) [inline docker-compose.yml]: Tiny comment here: Docker's console logging always includes the container tag, so the longer the naming is, the more space you lose off the left side.

Comment (hendricksond) [reply docker-compose.yml]: People still use tiny console windows?

Comment (grundoon) [reply docker-compose.yml]: Maybe I'm odd. Or just old. Or both. I still use 24 x 80 on a 27" display.

Comment (grundoon) [reply docker-compose.yml]: (not the *whole* display, mind you)

Comment (hendricksond) [reply docker-compose.yml]: Well, it's traditional. I stretch mine across most of my screen 😁

**Pull request merged**